### DISCUSSION TITLE
Create new entrie with API

### DISCUSSION THREAD

**Discussion by dtouffut** [NginxProxyManager/nginx-proxy-manager#1419] [Q&A]
Hello I love the product !
It's possible to create new proxy-hosts with the API?
@jc21

**Comment by chaptergy** [comment]
Please do not tag people in hopes they respond faster, it usually has the opposite effect.  
There is an API, but unfortunately it is not documented. See https://github.com/jc21/nginx-proxy-manager/issues/341#issuecomment-839129341

**Comment by DenAV** [comment]
Hello,
I tried to create a script.
Almost everything worked, only I can't find how to enter email for certbot.

https://github.com/DenAV/nginx-proxy-manager-ansible/blob/main/library/npm_proxy.py
a simple way to add a new proxy host via ansible playbook.
